kong.cloudformation.yaml

```diff
@@ -768,10 +795,11 @@ Resources:
                 {
                     missingok
                     notifempty
-                    size 500M
+                    size 5GB
```

Comment:
This is a very large file size. The default root volumes are 8GB.

Reply:
And the failures that happen when running out of disk space is an absolute nightmare 😢

Reply:
@calvinlfer @hackmad I set the limit to 1GB. Could you re-review?
The issue is that we are re-starting Nginx upon Log rotate so a size that is too small would cause us frequent Nginx restarts.